hrm_oo.py

```diff
@@ -0,0 +1,90 @@
+from readHR import EcgReader
+from hrmcalcs2oo import hrmcalcs
+from hrmtb import TachyBrady
+
+
+class HrmOutput:
+
+    def __init__(self, file='Test_ECG.csv', start_min=3, end_min=5,
+                 brady_limit=60, tachy_limit=100):
+        """
+
+        :param file: The file to make heart rate calculations for
+
+        :param start_min: User-inputted value for the "starting minute" of
+        average heart rate calculations, in minutes.
+
+        :param end_min: User-inputted value for the "ending minute" of average
+        heart rate calculations, in minutes.
+
+        :param brady_limit: The threshold value for bradycardia, with the
+        default set at 60 bpm
+
+        :param tachy_limit: The threshold value for tachycardia, with the
+        default set at 100 bpm
+        """
+        self.file = file
+        self.start_min = start_min
+        self.end_min = end_min
+        self.brady_limit = brady_limit
+        self.tachy_limit = tachy_limit
+        self.valid_file = False
+        self.peak_vector = []
+        self.instant_hr = []
+        self.average_hr = None
+        self.tachy = []
+        self.brady = []
+        if self.valid_file:
+            self.extract_vals()
+            self.print_hrmoutput()
```

Comment:
nit: instead of calling `print_hrmoutput()` in the init function, let the caller or user of your class call that function when/if they need it. Also, if "hrm" and "output" are separate words, you should snake_case 'em (`print_hrm_output`)

Reply:
I thought the point of our program is we want to be able to read in an ECG file and calculate heart rate and save it to a text file. This part is saving the text file so I thought we would want that to happen automatically, so taking it out of the init would make it an extra step. Should we change that and put it in the README file that you would need to run "test = hrm_oo(file)" to read in the file/perform calculations, and then "test.print_hrm_output()" in order to save?